edge-detect/coral/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.7'
+
+services:
+  coral-edge:
+    image: skyscan_fast_ai_image:latest
+    privileged: true
+    volumes:
+      - /dev/bus/usb:/dev/bus/usb
+      - /home/mobian/dphogan_test/edge_test/tosort:/data/tosort
```

Comment:
If this file is needed, it should include paths that will exist

Reply:
Renamed paths and clarified this file is demonstrative.